### PR TITLE
Issue #345: add missing translations of platform pages

### PR DESCRIPTION
**Pull request by lmillucci**
## Description

As already mentioned in #345, this PR tackles english translations of:
* IO
* OTELLO
* IndicePA
* NoiPA

And some minor fixes to the wording of italian version of IO and OTELLO platforms

## Checklist

- [x] I followed the indications in the [CONTRIBUTING](../CONTRIBUTING.md)
- [ ] The documentation related to the proposed change has been updated accordingly (also comments in code).
- [ ] Have you written new tests for your core changes, as applicable?
- [ ] Have you successfully ran tests with your changes locally?
- [x] Ready for review! :rocket:

### CODE REVIEW

File: _platforms/it/io.md [hunk 1]
```diff
@@ -16,17 +16,17 @@ github_team: io
 ## Intro
 
 io è l’applicazione del Governo italiano che permetterà a tutti i cittadini di usare servizi pubblici nazionali e locali dal proprio smartphone in maniera semplice, moderna e sicura.
-È un progetto Open Source che crescerà di giorno in giorno, coinvolgendo enti e servizi diversi.
+È un progetto Open Source che cresce di giorno in giorno, coinvolgendo enti e servizi diversi.
```
Comment (lmillucci) [inline _platforms/it/io.md:19]: Why the sentence was in the future? The project is growing everyday right now :)

File: _platforms/it/io.md [hunk 2]
```diff
@@ -16,17 +16,17 @@ github_team: io
...
 
-io mira ad agevolare la relazione tra i cittadini e la Pubblica Amministrazione (PA) attraverso la creazione di una piattaforma di componenti riutilizzabili in grado di rendere i servizi digitali più efficaci e migliorare la comunicazione tra i cittadini e la PA.
+io mira ad agevolare la relazione tra i cittadini e la Pubblica Amministrazione (PA) attraverso la creazione di una piattaforma di componenti riutilizzabili in grado di rendere i servizi digitali più efficaci.
```
Comment (lmillucci) [inline _platforms/it/io.md:21]: It was sounding bad in Italian. I think that a better relationship means a better communication without having to say it again in the same sentence

File: _platforms/it/io.md (hunk 1 above)
Comment (bfabio) [reply _platforms/it/io.md:19]: Hey @lmillucci, thanks for your work! Can you mention the changes to the Italian part as well in the commit log please?

Comment (lmillucci) [reply _platforms/it/io.md:19]: @bfabio done :+1:

File: _platforms/en/io.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+The preferences management API allow citizens to manage all the information associated with their digital citizen profile. Thanks to this information (communication channels, language preference, privacy settings, etc…) it is possible to create personalized digital services.
+
+The communication APIs provide a single channel that the Public Administration can use to send communications, notices and deadlines to the citizen. This communication system, together with the preferences management API, allows services to communicate with citizens on the basis of his own preferences.
+
+### How to contribute
+
+* [API Usage](https://teamdigitale.github.io/digital-citizenship/)
+* [GitHub Repository](https://github.com/teamdigitale/digital-citizenship-functions)
+* [How to contribute](https://github.com/teamdigitale/digital-citizenship/blob/master/CONTRIBUTING.it.md)
```
Comment (bfabio) [inline _platforms/en/io.md:37]:
```suggestion
* [API Usage](https://developer.io.italia.it/openapi.html)
* [GitHub Repository](https://github.com/pagopa/io-app)
* [How to contribute](https://github.com/pagopa/io/blob/master/CONTRIBUTING.it.md)
```

**Review by bfabio — CHANGES_REQUESTED** [review]

File: _platforms/en/io.md (hunk 3 above)
Comment (bfabio) [reply _platforms/en/io.md:37]: The links were fixed by #748 for the Italian version.

Comment (lmillucci) [reply _platforms/en/io.md]: fixed :+1:

**Review by bfabio — APPROVED** [review]

File: _platforms/en/indicepa.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+description: Database of national interest pursuant to Article 57-bis paragraph 1 of the Codice dell'Amministrazione Digitale.
+lang: en
+order: 4
+highlight: true
+comingsoon: false
+layout: platform
+resources:
+  - Documentation:
+    - title: Home page
+      url: http://indicepa.gov.it
```
Comment (bfabio) [inline _platforms/en/indicepa.md:15]:
```suggestion
      url: https://indicepa.gov.it
```

**Comment by bfabio** [comment]
Ok, this is embarassing. It was supposed to be merged but never was, I'm sorry @lmillucci.

I removed the IO changes because meanwhile the Italian page was modified.

**Pull request merged**